Makefile

```diff
@@ -71,6 +71,9 @@ run-test:
 e2e: kube-batch
 	hack/run-e2e.sh
 
+generate-code:
```

Comment:
I would suggest `generate-code: generate-deepcopy`, so developers can call only `make generate-code `

Reply:
The function of ``generate-deepcopy`` is already been included into ``generate-code``

Reply:
I noticed that, can i say the `generate-deepcopy` is a redundant or a legacy stuff, which can be removed?